### PR TITLE
doc(exercise 5): mint/burn AXL <-> Ethereum

### PR DESCRIPTION
**Pull request by ggutoski**

### CODE REVIEW

File: documentation/docs/Exercises/Exercise 5.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Axelar Network is a work in progress. At no point in time should you transfer an
 ## Prerequisites
 - Complete all steps from [Setup with Docker](/setup-docker) or [Setup with Binaries](/setup-binaries)
 - Have a Ethereum wallet setup and have an Ethereum address funded with some Ether (You can also choose to use the [Chrome plugin](https://chrome.google.com/webstore/detail/mew-cx/nlbmnnijcnlegkjjpcfjclmcfggfefdm?hl=en))
-- Your Axelar Network address have`satoshi` and `uphoton (ibc/287EE075B7AADDEB240AFE74FA2108CDACA50A7CCD013FA4C1FCD142AFA9CA9A)` balances. If you do not have these assets, please follow `Exercise 3` and `Exercise 4` to mint some.
+- Your Axelar Network address has a positive `uaxl` balance. You can get AXL tokens from the [Axelar faucet](http://faucet.testnet.axelar.dev/).
```
Comment (Windforcer) [inline documentation/docs/Exercises/Exercise 5.md:21]: positive sounds weird to me. Maybe "You have some uaxl tokens on your Axelar Network address" or something similar?

File: documentation/docs/Exercises/Exercise 5.md [hunk 2]
```diff
@@ -33,50 +33,34 @@ Axelar Network is a work in progress. At no point in time should you transfer an
...
- denom: satoshi
 - amount: "1000000"
  denom: uaxl
-- amount: "1000000"
- denom: ibc/287EE075B7AADDEB240AFE74FA2108CDACA50A7CCD013FA4C1FCD142AFA9CA9A
 ```
 2. Create a deposit address on Axelar Network (to which you'll deposit coins later)
 
-[token] is one of `uaxl`, `satoshi`, and `uphoton`
+[token] is `uaxl`.
```
Comment (Windforcer) [inline documentation/docs/Exercises/Exercise 5.md:53]: Should we add Terra uusd token? I think it replaced uphoton

**Review by Windforcer — APPROVED** [review]

File: documentation/docs/Exercises/Exercise 5.md (hunk 2 above)
Comment (ggutoski) [reply documentation/docs/Exercises/Exercise 5.md:53]: > Should we add Terra uusd token? I think it replaced uphoton

Makes sense to me.  I just did what I was told. 😛 @haiyizxx any opinion here?

Comment (haiyizxx) [reply documentation/docs/Exercises/Exercise 5.md:53]: ya, fine to remove. We do UST from Terra to EVM directly in exercise 4 now, exercise 5 is for AXL from Axelar to EVM.

**Pull request merged**